go.mod

```diff
@@ -7,28 +7,26 @@ require (
 	github.com/go-logr/logr v0.2.1
 	github.com/go-logr/zapr v0.2.0 // indirect
 	github.com/google/go-cmp v0.5.2
-	github.com/google/uuid v1.1.2 // indirect
 	github.com/manifestival/controller-runtime-client v0.3.0
 	github.com/manifestival/manifestival v0.6.1
 	github.com/openshift/api v0.0.0-20200930075302-db52bc4ef99f
 	github.com/openshift/client-go v0.0.0-20200929181438-91d71ef2122c
-	github.com/openzipkin/zipkin-go v0.2.5 // indirect
 	github.com/operator-framework/api v0.3.16
 	github.com/operator-framework/operator-lifecycle-manager v0.0.0-20201013052701-d7b2aff003ad
 	github.com/operator-framework/operator-sdk v0.19.4
 	github.com/prometheus-operator/prometheus-operator v0.43.0
 	github.com/prometheus/client_golang v1.8.0
 	github.com/spf13/pflag v1.0.5
-	go.uber.org/zap v1.15.0
+	go.uber.org/zap v1.16.0
 	google.golang.org/genproto v0.0.0-20200914193844-75d14daec038 // indirect
 	k8s.io/api v0.19.2
 	k8s.io/apimachinery v0.19.2
 	k8s.io/client-go v12.0.0+incompatible
-	knative.dev/eventing v0.18.5-0.20201105155307-650096a39064
+	knative.dev/eventing v0.19.2
```

Comment:
This should not be done via go.mod but **has** to be done via updating `hack/update-deps.sh`. We should probably add a check to the actions that prevent manual go.mod mucking for individual projects that should be bumped as a whole.